### PR TITLE
Follow unacknowledged write with a find in spec tests

### PR DESCRIPTION
**Pull request by jyemin**
This will force completion of the w: 0 write, which addresses a TCP-based race condition in the tests

JAVA-5398

### CODE REVIEW

**Review by stIncMale — APPROVED** [review]

**Comment by stIncMale** [comment]
Why is the PR named "WIP", is there something that it is currently missing?

**Comment by jyemin** [comment]
Sorry, it was WIP while the test changes were going through code review in the specifications repo.  I updated the title and description, and will edit the commit message during the merge process.

**Comment by stIncMale** [comment]
> I updated the title

It's still "WIP"

File: driver-core/src/test/resources/unified-test-format/command-logging/unacknowledged-write.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "description": "command-logging",
-  "schemaVersion": "1.13",
+  "description": "unacknowledged-write",
```
Comment (jyemin) [inline driver-core/src/test/resources/unified-test-format/command-logging/unacknowledged-write.json:2]: This line was from a previous change to the file

File: driver-core/src/test/resources/unified-test-format/command-monitoring/unacknowledgedBulkWrite.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "description": "unacknowledgedBulkWrite",
-  "schemaVersion": "1.0",
+  "schemaVersion": "1.7",
```
Comment (jyemin) [inline driver-core/src/test/resources/unified-test-format/command-monitoring/unacknowledgedBulkWrite.json:3]: Required for `ignoreExtraEvents`

File: driver-core/src/test/resources/unified-test-format/command-logging/unacknowledged-write.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+  "schemaVersion": "1.16",
```
Comment (jyemin) [inline driver-core/src/test/resources/unified-test-format/command-logging/unacknowledged-write.json:3]: Required for `ignoreExtraMessages`

**Review by stIncMale — APPROVED** [review]

**Pull request merged**